### DISCUSSION TITLE
Is private developer app supposed to say "Current edit is saved as draft"

### DISCUSSION THREAD

**Discussion by JeffMash** [amzn/selling-partner-api-models#3478] [General] [answered]
One of the issues I'm dealing with (when trying to call a function in the library) is a "403 Forbidden" error (see Discussion amzn/selling-partner-api-models#2821)

Anyway, I saw some other posts that say this COULD be caused by the fact that the app is in a draft state.  I am developing this app just for our private use (to update our own inventory and sync it with our website), so I don't need it registered to any App Store or anything like that.

1.  Can we still use the jlevers library if the app says "Current edit is saved as draft"?
2.  If not, and if it needs to be published, then how do we do that?

This is what mine looks like right now (screenshot):

https://i.postimg.cc/MTZvB6LF/screenshot-418.png

If anyone can help me with this (or Discussion amzn/selling-partner-api-models#2821), I will be greatly appreciated and love you forever.  <3

**Comment by JeffMash** [comment, marked as answer]
From what I can gather, I think it's okay that the App says "Current Edit Saved as a Draft" if you're only going to be a private developer without publishing anything to the AppStore.   If I'm incorrect on this, please let m know.

**Reply by Akum777** [reply]
yes this is okay for private developer applications.
